Dockerfile

```diff
@@ -0,0 +1,19 @@
+# mbc-transactional-email
+
+FROM dosomething/mb:latest
+MAINTAINER Sergii Tkachenko <sergii@dosomething.org>
+
+ARG COMPOSER_GITHUB_TOKEN
+ENV RABBITMQ_HOST=192.168.99.10\
```

Comment:
I don't know enough about Docker so perhaps this is a silly question but will the IP address always be this value?